node_modules/oae-core/about/about.html

```diff
@@ -0,0 +1,48 @@
+<!-- CSS -->
+<link rel="stylesheet" type="text/css" href="css/about.css" />
+
+<div class="about-widget">
+    <div id="about-header"><!-- --></div>
+    <div id="about-description"><!-- --></div>
+    <div id="about-members"><!-- --></div>
+</div>
+
+<div id="about-header-template"><!--
+    ${header('__MSG__ABOUT__', action)}
+--></div>
+
+<div id="about-description-template"><!--
+    {if contextProfile.description}
+        <div class="about-description-content">
+            ${oae.api.util.security().encodeForHTMLWithLinks(contextProfile.description).replace(/\n/g, '<br/>')}
+        </div>
+    {else}
+        <div class="about-description-empty">
+            {if contextProfile.isManager}
+                <a class="group-trigger-editgroup" data-editgroup-focus="#editgroup-description" href="#" title="__MSG__ADD_AN_INTRODUCTION_TO_THIS_GROUP__">
+                    __MSG__ADD_AN_INTRODUCTION_TO_THIS_GROUP__
```

Comment:
This should be `ADD_A_DESCRIPTION_TO_THIS_GROUP`. The title attribute doesn't need to be set as it doesn't provide any extra information on top of the element's text